packages/graphql/src/schemaTypes/objectTypes/gql-Subscription.ts

```diff
@@ -17,5 +17,12 @@ export const Subscription = subscriptionType({
       subscribe: (source, args, ctx) => ctx.emitter.subscribeTo('devChange'),
       resolve: (source, args, ctx) => ctx.coreData.dev,
     })
+
+    t.field('browserStatusChange', {
+      type: Query,
```

Comment:
We could prob make `CurrentProject` as the type here since that's what it's scoped to

Reply:
Done!